src/cgame/rocket/rocket_documents.cpp

```diff
@@ -162,6 +162,8 @@ void Rocket_DocumentAction( const char *name, const char *action )
 		{
 			Rml::String url = document->GetSourceURL();
 			document->Close();
+			Rml::Factory::ClearStyleSheetCache();
```

Comment:
Ah, I thought i explained in the commit message. RmlUI will cache stylesheets, so if you want to rapidly iterate on RCSS changes, you can't just make changes and run `/rocket <document> reload` since it will used the old cached versions.

Reply:
Could you use FS::PakPath::LocateFile and clear the cache only if it comes from a dpkdir?

Reply:
Not without parsing the file. For example, if I run `rocket menu_main reload`, It will only reload *that* file. Say `menu_main.rml` contains `<link type="text/rcss" href="/ui/shared/basics.rcss" />`, and I edit `basics.rcss`, any changes to `basics.rcss` will not be reflected in the reload.

Given that this is a developer focused command and there really isn't a huge performance penalty for this, I think it is a safe change.

Reply:
Oh, I didn't see that this is part of the "reload" command. It makes sense to reload things in that case...